pymc/sampling/mcmc.py

```diff
@@ -481,11 +479,23 @@ def sample(
         model.check_start_vals(ip)
         _check_start_shape(model, ip)
 
+    # Create trace backends for each chain
+    traces = [
+        _init_trace(
+            expected_length=draws + tune,
+            stats_dtypes=step.stats_dtypes,
+            chain_number=chain_number,
+            trace=trace,
+            model=model,
+        )
+        for chain_number in range(chains)
+    ]
+
     sample_args = {
         "draws": draws,
         "step": step,
         "start": initial_points,
-        "trace": trace,
+        "traces": traces,
```

Comment:
Is potential breakage worth changing the arg to plural?

Reply:
It's a signature change regardless: `Optional[BaseTrace]` → `Sequence[BaseTrace]` for multiple functions that this dict is fed to.

All of these are internal API though, so breakage should not be a problem